packages/x-charts/src/internals/domUtils.ts

```diff
@@ -98,6 +98,11 @@ export const getStyleString = (style: React.CSSProperties) =>
     );
 
 let domCleanTimeout: NodeJS.Timeout | undefined;
+const domResults = { count: 0, time: 0 };
+if (typeof window !== 'undefined') {
+  window.domResults = domResults;
+}
```

Comment:
Rudimentary way to check how long we're spending in text measurements.

By running `domResults`, you should see how many times (`count`) and how long it took in ms (`time`) to measure all the text that was measured so far. These values only include the cache misses.